data/ButterflyPACK/e4s.yaml

```diff
@@ -0,0 +1,5 @@
+- e4s_product: butterflypack
+  docs:  [README.md, LICENSE]
```

Comment:
There is no LICENSE file in this repo. Should this be license.txt instead? Also note that if you create a .e4s folder in your project repo and store your e4s.yaml file in there your repo's copy will take precedence over the copy in the docportal repo.

Reply:
@wspear Thanks for reviewing this! I've corrected the license file. I also created a .e4s/e4s.yaml file in the code repo. Can you take a look at this again?